OpenRA.Mods.Common/Traits/TransformOnCrush.cs

```diff
@@ -0,0 +1,57 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public class TransformOnCrushInfo : ITraitInfo
+	{
+		[ActorReference] public readonly string IntoActor = null;
```

Comment:
Hm, do we want this to have a `Fieldloader.Require` (we're not very consistent with that)?

Reply:
Good idea. I assume it will crash if not set. Done.